index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome Page</title>
+    <link rel="stylesheet" href="/css/style.css">
+</head>
+<body>
```

Comment:
what about putting all elements directly in the body.
then make the body  column flex
after centering the items, there will be no need for margins left and right
and it will be appropriate for desktop